datasets/ehealth_kd/ehealth_kd.py

```diff
@@ -0,0 +1,186 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The eHealth-KD 2020 Corpus."""
+
+from __future__ import absolute_import, division, print_function
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{overview_ehealthkd2020,
+  author    = {Piad{-}Morffis, Alejandro and
+               Guti{\'{e}}rrez, Yoan and
+               Cañizares-Diaz, Hian and
+               Estevez{-}Velarde, Suilan and
+               Almeida{-}Cruz, Yudivi{\'{a}}n and
+               Muñoz, Rafael and
+               Montoyo, Andr{\'{e}}s},
+  title     = {Overview of the eHealth Knowledge Discovery Challenge at IberLEF 2020},
+  booktitle = ,
+  year      = {2020},
+}
+"""
+
+_DESCRIPTION = """\
+Dataset of the eHealth Knowledge Discovery Challenge at IberLEF 2020. It is designed for
+the identification of semantic entities and relations in Spanish health documents.
+"""
+
+_HOMEPAGE = "https://knowledge-learning.github.io/ehealthkd-2020/"
+
+_LICENSE = "https://creativecommons.org/licenses/by-nc-sa/4.0/"
+
+_URL = "https://raw.githubusercontent.com/knowledge-learning/ehealthkd-2020/master/data/"
+_TRAIN_DIR = "training/"
+_DEV_DIR = "development/main/"
+_TEST_DIR = "testing/scenario3-taskB/"
+_TEXT_FILE = "scenario.txt"
+_ANNOTATIONS_FILE = "scenario.ann"
+
+
+class EHealthKD(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class EhealthKD(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `ehealth_kd`